tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -280,7 +280,46 @@ def restart_thermal_control_daemon(dut):
     config_reload(dut)
     assert 0, 'Wait thermal control daemon restart failed'
 
+def stop_thermal_control_daemon(dut):
+    """
+    Stop thermal control daemon
+    """
+    logging.info('Stopping thermal control daemon on {}...'.format(dut.hostname))
+    find_thermalctld_pid_cmd = 'docker exec -i pmon bash -c \'pgrep -f thermalctld\' | sort'
+    output = dut.shell(find_thermalctld_pid_cmd)
+    assert output["rc"] == 0, "Run command '%s' failed" % find_thermalctld_pid_cmd
+    assert len(output["stdout_lines"]) >= 2, "There should be at least 2 thermalctld process"
+                                                                    
+    stop_thermalctl_cmd = "docker exec -i pmon bash -c 'supervisorctl stop thermalctld'"
+    output = dut.shell(stop_thermalctl_cmd)
+    if output["rc"] == 0:
+        assert len(output["stdout_lines"]) >= 1, "There should be at least 1 thermalctld process"
+        logging.info("thermalctld processes stopped successfully on {}".format(dut.hostname))
+        return
+    # try restore by config reload...
+    config_reload(dut)
```

Comment:
We needn't do config reload here, as config reload will restart the dockers again and will start the thermal control daemon. (which is not we want in this "stop" thermal control daemon API)

Reply:
thanks Judy, will change this